frontend/lib/screens/browse_events.dart

```diff
@@ -0,0 +1,116 @@
+// Browse events screen
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import '../widgets/nav_drawer.dart';
+import '../widgets/register_button.dart';
+
+class BrowseEvents extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // get a list of all events from backend
+    final list = [
+      {
+        "name": 'Martha\'s Birthday Bash',
+        "date": "2016-07-20T12:00:15+00:00",
+        "tag": "Meeting",
+        "organizer": "Martha's Family",
+        "location": "1234 W Sample St, Vancouver, BC"
+      },
+      {
+        "name": 'Martha\'s Birthday Bash',
+        "date": "2016-07-20T12:00:15+00:00",
+        "tag": "Meeting",
+        "organizer": "Martha's Family",
+        "location": "1234 W Sample St, Vancouver, BC"
+      }
+    ];
```

Comment:
Could this have been a `yaml` file instead?

Reply:
Moved this to a json file in commit 029e4c6f432a732625dbf521275750643f5e2128